### PR TITLE
Add Testing for Windows, ubuntu, and macOS.

### PR DESCRIPTION
**Pull request by UrielCh**
All test Pass 🥇  

But for now you will have to keep them passing :grin:

### CODE REVIEW

File: lib/ffi.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import { join } from "https://deno.land/std@0.170.0/path/mod.ts";
+// can be use to improve error handling but requiere extra --allow-read permission
```
Comment (aapoalas) [inline lib/ffi.ts]: `requires`

Comment (UrielCh) [reply lib/ffi.ts]: Fixed in the last commit.

File: lib/ffi.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import { join } from "https://deno.land/std@0.170.0/path/mod.ts";
+// can be use to improve error handling but requires extra --allow-read permission
+// import { existsSync } from "https://deno.land/std@0.170.0/fs/mod.ts";
```
Comment (aapoalas) [inline lib/ffi.ts:3]: Remove these lines: I'll improve the error handling by checking if the error messages... Eventually :)

File: lib/ffi.ts [hunk 3]
```diff
@@ -10,6 +13,19 @@ import * as FatalErrorHandler from "./include/FatalErrorHandler.h.ts";
 import * as Index from "./include/Index.h.ts";
 import * as Rewrite from "./include/Rewrite.h.ts";
 
+/**
+ * Windows dll have a few missing symbols
+ * windows dll come from `choco install --version 14.0.6 llvm`
+ * md5 59beb52cef40898b0f24cdffc6cf2984
+ * `dumpbin /exports libclang.dll`
+ */
+export const WINDOWS_MISSING_SET = [
```
Comment (aapoalas) [inline lib/ffi.ts]: No need to export these now.

**Comment by aapoalas** [comment]
Closing the extra PRs.

**Pull request closed without merging**